client/rest/rest.go

```diff
@@ -0,0 +1,26 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// addHTTPDeprecationHeaders is a mux middleware function for adding HTTP
+// Deprecation headers to a http handler
+func addHTTPDeprecationHeaders(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Deprecation", "true")
```

Comment:
Do we also want to set `Sunset` and or `Warning` headers?

Reply:
`Sunset` headers requires a specific date of sunset, I can add `Warning` headers with essentially the same information, but a bit more verbose.